tests/DwitTech.AccountService.WebApi.Tests/Controllers/UserControllerTest.cs

```diff
@@ -353,5 +353,81 @@ public async Task Logout_Should_Return_False_If_AuthorizationHeader_Is_Not_Given
             Assert.IsType<BadRequestObjectResult>(result);
 
         }
+
+        [Fact]
+        public async Task DeleteUser_Returns_NoContentResult()
```

Comment:
I think you forgot to add "WhenSuccessful" in your method name here.

Reply:
> I think you forgot to add "WhenSuccessful" in your method name here.

Resolved